docs/modules/hdfs/examples/getting_started/hdfs.yaml

```diff
@@ -11,13 +11,13 @@ spec:
     dfsReplication: 1
   nameNodes:
     config:
-      listenerClass: cluster-internal
+      listenerClass: external-stable
```

Comment:
why is this needed?

Reply:
It is not strictly needed, but I would argue users want to able to access the stuff the just deployed ;)
And port-forwards IMHO should be a thing of last resort as they break all the time.

In this concrete HDFS case users accessing the WebUIs, for Trino and Superset it would be submitting queries or creating Dashboards,

Reply:
I was confused when debugging  because this particular guide doesn't access HDFS from outside the cluster. Also the doc doesn't explain what the `listenerClass` even does (which is fine in this context) , that's why I changed it to the "safer" value.

Reply:
I see. The default is actually `cluster-internal` for security reasons, so we need to set it explicitly in case we want external access.

Would you be in favor of internal or external access here?

Reply:
> Would you be in favor of internal or external access here?

In the current state of the guide I would prefer "internal". If the guide starts explaining the listener class and/or adds instructions on how to access the cluster from outside, then yes, change it to "external".

Reply:
I think we need some sort of strategy/guideline what we want to do, created https://github.com/stackabletech/decisions/issues/7. Happy about comments!